pkg/openwsn/contrib/leds.c

```diff
@@ -86,7 +87,7 @@ static void _blink_checked(gpio_t pin)
         /* toggle for ~10s if ztimer is used */
         for (uint8_t i = 0; i < 100; i++) {
             _toggle_checked(pin);
-            for (uint32_t i = 0; i < (CLOCK_CORECLOCK / 50); i++) {
+            for (uint32_t i = 0; i < (coreclk() / 50); i++) {
```

Comment:
I assume it is guaranteed that coreclk() always is either a read of a non-volatile global or a const; but somehow when i read this i assume there might be a function call and a beginner might be inspired by this. ->
```suggestion
            uint32_t runs = coreclk() / 50;
            for (uint32_t i = 0; i < runs; i++) {
```